php/helpers.php

```diff
@@ -151,82 +151,6 @@ function block_field_config( $name ) {
 	return (array) $block_lab_config->fields[ $name ];
 }
 
-/**
```

Comment:
Now, It'd be good to only use this file for functions that other plugins or themes might need access to, like `block_field()`.